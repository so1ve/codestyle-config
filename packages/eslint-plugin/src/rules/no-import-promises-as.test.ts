import { run } from "./_test";
import rule, { RULE_NAME } from "./no-import-promises-as";

const valid = ['import { promises } from "fs";'];
const invalid = [
	'import { promises as fs } from "fs";',
	'import { promises as fs } from "node:fs";',
];

run({
	name: RULE_NAME,
	rule,
	valid,
	invalid,
});
