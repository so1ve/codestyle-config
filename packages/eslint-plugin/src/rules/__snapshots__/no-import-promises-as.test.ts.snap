// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`no-import-promises-as > invalid > import { promises as fs } from "fs"; 1`] = `
"import {  } from "fs";
import fs from "fs/promises";"
`;

exports[`no-import-promises-as > invalid > import { promises as fs } from "node:fs"; 1`] = `
"import {  } from "node:fs";
import fs from "node:fs/promises";"
`;
